src/main/java/frc/robot/subsystems/elevator/ElevatorInputs.java

```diff
@@ -0,0 +1,15 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Velocity;
+import org.littletonrobotics.junction.AutoLog;
+
+@AutoLog
+public class ElevatorInputs {
```

Comment:
Should also log limit switches state.

Reply:
eaf49ea3d4d6b5370f8688873f108a0e38f41ad9